README.md

```diff
@@ -1260,6 +1260,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 
 - ### **J**
 
+  - [Jetur Gavli](https://github.com/jeturgavli)
```

Comment:
Please add your name at the right position to keep this list in the correct alphabetical order. (It should be in line 1313.)

Reply:
I have added my profile according to alphabetical order. And it has been fixed.  Now my profile is on line 1313.